### DISCUSSION TITLE
Looking for AS3 Developers with Starling and Nape Physics Experience for an Exciting Adobe AIR Game Develpment

### DISCUSSION THREAD

**Discussion by Sephius-Fernando** [airsdk/Adobe-Runtime-Support#3440] [General]
![BroadcastHeader](https://github.com/user-attachments/assets/3c96713b-6a1c-48ab-abfd-5322409553fc)

Hello fellow Adobe Air Developers

I'm reaching out to this community in search of AS3 developers with experience in Starling and Nape Physics. We're working on a project in development for quite some time called The Light of the Darkness: ORIGINS, an Action RPG that we're building with Adobe AIR using a proprietary engine. 

A trailer of the game can be seen here: 
https://youtu.be/htX6Co7cxRQ?si=oY61dKz7vXohERwu

The Steam page can be seen here:
https://store.steampowered.com/app/2490890/The_Light_of_the_Darkness_Origins/

Our goal is to create a game that blends the exploration and progression mechanics of Castlevania with the narrative depth of Legacy of Kain. We believe that Adobe AIR, combined with tools like Starling and Nape Physics, still has much potential to offer for game development, and we want to showcase that with our project.

We know there are still many talented developers here who share a love for Adobe AIR and AS3, and we're hoping some of you might be interested in collaborating with us in order to finish the development. We're looking for developers who are passionate about what they do and who want to help us push the boundaries of what's possible with this technology.

If you're interested in joining us or have any suggestions on where else we might find developers with these skills, or help us with insights and support, we would love to hear from you. Any advice or connections would be greatly appreciated!

Thank you for taking the time to read this, and we hope to hear from some of you soon!

**Comment by marchbold** [comment]
Looks like an amazing project! I'd be happy to be involved. I have helped lots of game developers particularly with service integrations (IAP, game services, advertising etc). Feel free to reach out to me via ma at distriqt.com

**Reply by Sephius-Fernando** [reply]
Thanks. We you make contact! Do you have expertise in Steam integration in adobe air projects?

**Reply by marchbold** [reply]
We've done some basic integration with steam, would love to dive deeper into it though!

**Comment by Devin-Licastro** [comment]
Looks super cool! Your background art is especially stunning. I've wish-listed it!

**Reply by Sephius-Fernando** [reply]
Thanks @Devin-Licastro !

**Reply by raresn** [reply]
Same here :)

**Comment by ylazy** [comment]
Subscribed!

Really love the background art & effects... make me feel like I want to live there. I wish I could draw something like that, but I know I can't because I have no talent for drawing/designing. I can't make beautiful games, so I decided to make beautiful applications with Adobe Air:

![image](https://github.com/user-attachments/assets/eaf6fe38-1374-4b61-a379-b4f8e5729a41)

And I'm working hard to make it happens on one day soon... maybe in early 2025, when [Adobe Spectrum 2](https://s2.spectrum.adobe.com/) is ready.

> We know there are still many talented developers here who share a love for Adobe AIR and AS3

I'm still using Flash for presenting, using Air to create tools for working, and education applications. For most applications - when the performance is not too critical, then Air is the best choice for me. The best syntax, the simple but powerful APIs, the acceptable performance, the best support for vector-based content, for animation & rich-media content... It can only be AS3, not Java, not C#, not JS/TS for sure, not any other programming languages.

![image](https://github.com/user-attachments/assets/71736a7b-76b1-47ce-bb27-d4528b87fee0)

I still remembered the day when I haven't known anything about programming, when I drew a stickman, gave him a name, clicked on a code snippet, ctrl+enter, and I can drag-drop him from place to place... That's... amazing!!! I feel like I'm a God just created something!

I hope that one day we can [bring Flash back](https://ruffle.rs/) and some day AS3 can be one of top programming languages again.
